tests/collapse.spec.ts

```diff
@@ -0,0 +1,61 @@
+import { test, expect } from '@playwright/test';
+
+test('Collapse E2E Tests', async ({ page }) => {
+  await page.goto('/test.html');
+
+  const collapse = page.getByTestId('collapse')
```

Comment:
Id recommend changing the variable name to collapseDefault or something for clarity